src/main/java/com/ibm/watson/apis/conversation_enhanced/discovery/DiscoveryQuery.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * (C) Copyright IBM Corp. 2016. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.ibm.watson.apis.conversation_enhanced.discovery;
+
+import com.ibm.watson.apis.conversation_enhanced.utils.Constants;
+import com.ibm.watson.developer_cloud.discovery.v1.Discovery;
+import com.ibm.watson.developer_cloud.discovery.v1.model.query.QueryRequest;
+import com.ibm.watson.developer_cloud.discovery.v1.model.query.QueryResponse;
+
+public class DiscoveryQuery {
+
+	private String userName;
+	
+	private String password;
+	
+	private String collectionId;
+	
+	private String environmentId;
+	
+
+	//private static final Logger logger = LogManager.getLogger(DiscoveryQuery.class.getName());
+
+	public DiscoveryQuery() {
+		userName = System.getenv("DISCOVERY_USERNAME");
+		password = System.getenv("DISCOVERY_PASSWORD");
+		collectionId = System.getenv("DISCOVERY_COLECTION_ID");
+		environmentId = System.getenv("DISCOVERY_ENVIROMENT_ID");
+	}
+
+	/**
+	 * Use the Watson Developer Cloud SDK to send the user's query to the
+	 * discovery service
+	 * 
+	 * @param userQuery
+	 *            The user's query to be sent to the discovery service
+	 * @return The query responses obtained from the discovery service
+	 * @throws Exception
+	 */
+	public QueryResponse query(String userQuery) throws Exception {
+
+		// Configure the Watson Developer Cloud SDK to make a call to the
+		// appropriate discovery
+		// service. Specific information is obtained from environment variable
+		// and the services
+		// associated with the app. See the Query constructor for details.
+
+		Discovery discovery = new Discovery(Constants.DISCOVERY_VERSION);
+		discovery.setEndPoint(Constants.DISCOVERY_END_POINT);
+		discovery.setUsernameAndPassword(userName, password);
+
+
+		QueryRequest.Builder queryBuilder = new QueryRequest.Builder(environmentId, collectionId);
+		queryBuilder.query("searchText:" + userQuery + ",enrichedText:" + userQuery);
```

Comment:
Use `String.format()` instead of concatenate String. You can also use `StringBuffer` or `StringBuilder`